src/partisan_config.erl

```diff
@@ -37,25 +37,31 @@ set(Key, Value) ->
 peer_config() ->
     Port = case partisan_config:get(peer_port, undefined) of
         undefined ->
-            %% Generate a random peer port.
-            rand_compat:seed(erlang:phash2([node()]),
-                             erlang:monotonic_time(),
-                             erlang:unique_integer()),
-            RandomPeerPort = rand_compat:uniform(1000) + 10000,
+            case application:get_env(partisan, peer_port) of
```

Comment:
Yea, I could certainly change `get/2` to have this functionality instead, making it available to all config variables.